tests/overlay/test

```diff
@@ -17,8 +17,8 @@ sub cleanup() {
 
 sub getfilecon {
         my ($filename) = @_;
-	$output = `ls -dZ $filename | cut -f1 -d' '`;
-	$output =~ s/\n//g;
+	$output = `getfattr -d -m security.selinux $filename --only-values 2> /dev/null`;
```

Comment:
Why -m instead of -n? -m is for patterns where you want to match more than one xattr name.  -n is for a specific, fully-specified xattr name.